arch/z80/src/ez80/Toolchain.defs

```diff
@@ -66,9 +66,19 @@ ARCHASMWARNINGS = -W
 
 # Optimization of unused sections
 
+ifneq ($(CONFIG_DEBUG_NOOPT),y)
```

Comment:
there is `ifeq ($(CONFIG_DEBUG_NOOPT),y)` at line 45. Please merge with that selection.
Or maybe you can add `CONFIG_DEBUG_CUSTOMOPT` and make it the same way as in other files

Reply:
Done.